src/index.ts

```diff
@@ -126,29 +126,44 @@ export const createInstance = (config: Config): Instance => {
 
       return tx;
     },
+    getDomainMetadata: async (
```

Comment:
This doesn't seem to be included in `types.ts` on the definition (interface) of a zNS SDK Instance